### PR TITLE
Event permissions must match resource permissions

### PR DESCRIPTION
**Pull request by codekultur**
Fixed https://github.com/zalando/restful-api-guidelines/issues/171

### CODE REVIEW

File: events/events.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ There are cases where it could make sense to define data change events that don'
  - Events that are the result of a computation, such as a matching algorithm, or the generation of enriched data, and which might not be stored as entity by the service. 
 
 
+## {{book.must}} Permissions on events must correspond to API permissions
+If a resource can be accessed synchronously via a REST API and asynchronously via an event, the same permission must apply to protect access to the resource.
```
Comment (ePaul) [inline events/events.md]: Possibly mention _read_ accesses here?

**Comment by ePaul** [comment]
Discussed in the meeting: Add a bit of rationale.

File: events/events.md (hunk 1 above)
Comment (rcillo) [inline events/events.md]: Is this about [read scopes](https://github.com/zalando/nakadi/blob/ad6193782718af324be4f603571fb7b18dae8d2f/api/nakadi-event-bus-api.yaml#L1774-L1781) ?

Comment (ePaul) [inline events/events.md]: @rcillo Yes. It might be a good idea to link this.

**Comment by tkrop** [comment]
Also it is not telling the whole story, it is a good start. Lets move on with this.

**Comment by tkrop** [comment]
👍

**Comment by ePaul** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Pull request merged**